docs/TrialAccounts/offboarding.md

```diff
@@ -0,0 +1,180 @@
+
+### Claiming
+
+As per the official linkdrop standard, there are 3 methods that are important for claiming linkdrops:
+
+```ts
+/// Returns the KeyInfo associated with a given public key
+/// Panics if the key does not exist
+function get_key_information(key: string) -> KeyInfo
+
+/// Transfer assets linked to a given public key to an `account_id`.
+///
+/// Arguments:
+/// * `account_id` the account that should receive the linkdrop assets.
+///
+/// Returns `true` if the claim was successful meaning all assets were sent to 
+/// the `account_id`.
+function claim(account_id: string) -> Promise<boolean>
+
+
+
+/// Creates a new NEAR account and transfers all assets linked to a given public 
+/// key to the *newly created account*.
+/// 
+/// Arguments:
+/// * `new_account_id`: the valid NEAR account which is being created and should 
+///   receive the linkdrop assets
+/// * `new_public_key`: the valid public key that should be used for the access 
+//    key added to the newly created account.     
+///
+/// Returns `true` if the claim was successful meaning the `new_account_id` was 
+/// created and all assets were sent to it.
+function create_account_and_claim(new_account_id: string, new_public_key: string) -> Promise<boolean>
+```
+
+For an arbitrary linkdrop (a URL containing the contract and secret key), the flow is as follows:
+1. The user clicks a link or scans a QR code representing the linkdrop.
+2. The app calls `get_key_information` to check what's in the linkdrop.
+    - If the linkdrop was already claimed (or doesn't exist), the call will panic and the frontend indicates that the drop is invalid or claimed.
+3. Depending on what is returned from the call, the app should display what assets the user is about to claim (e.g NFTs, FTs, $NEAR etc.)
+4. The frontend exposes a form field and the user either inputs a new account or an existing one.
+5. The frontend calls `claim` or `create_account_and_claim` depending on the user's input. 
+6. The linkdrop is claimed and the assets are sent to the user's account.
+
+### Trial Account Linkdrops
+
+Now that the basics of linkdrop claiming have been covered, you'll look at how trial accounts can be treated as a linkdrop. From above, there are a few requirements that need to be met:
+- The URL should contain a contract and secret key
+- The contract should implement the `get_key_information` method so that frontends know when a drop is valid.
+- The contract should implement the `create_account_and_claim` method for claiming.
+
+The `new_public_key` that is passed into `create_account_and_claim` is added as the account's full access key and is generated on a per-app basis. For example, MyNEARWallet generates the key using a seedphrase while FastAuth generates it using biometrics. The trial account contract can use `create_account_and_claim` to perform the offboarding logic. It should take in the new public key and do the following (assuming the account can exit):
```

Comment:
where is `create_account_and_claim` being implemented? i think this circles back to my original comment at line 76 regarding not know which contract you're modifying